### PR TITLE
Use QDebug category when we build with qt5

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

**Comment by Montel** [comment]
Do you have some reviews on this patch ?

**Comment by pipacs** [comment]
What is the need/use case this patch is trying to address?

**Comment by Montel** [comment]
Disable noise when we use in project.
in qt5 we can do it so it can be useful otherwise we will have a lot of debug .

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ set( o2_SRCS
     o2simplecrypt.cpp
     o0settingsstore.cpp
     o0baseauth.cpp
+    o2debug.cpp
```
Comment (pipacs) [inline src/CMakeLists.txt]: Prefer o0debug for the module and function name.

**Comment by pipacs** [comment]
Makes sense then. Could the code be a bit more symmetric though: 

* Call the module o0debug.h/cpp
* Call the macros o0Debug() and o2Warning() (don't forget to add the parentheses)

**Comment by Montel** [comment]
"Call the macros o0Debug() and o2Warning() (don't forget to add the parentheses)"
We can't add parentheses
as in qt4 we need to use qDebug()
and in qt5 we need to use qCDebug(O2_DEBUG_LOG)
so it's not possible to add parentheses here

**Comment by pipacs** [comment]
Shouldn't ```#define o0Debug() qcDebug(O2_DEBUG_LOG)``` do the trick?

**Pull request closed without merging**